test/Microsoft.Health.Dicom.Web.Tests.E2E/Functions/AzureFunctionsConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Memory;
+
+namespace Microsoft.Health.Dicom.Web.Tests.E2E.Functions
+{
+    public static class AzureFunctionsConfiguration
```

Comment:
I'll eventually move a bunch of this into its own test package for healthcare shared components.